### PR TITLE
Update readme with better explanations

### PR DESCRIPTION
**Pull request by markomitranic**
The project goals and effects have changed since its creation, thus i felt it was time to slightly update the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,15 +31,23 @@ D4M is slow. Primarily because of its osxfs/grpcfuse sharing filesystem. On the
 There are various ways to code in this setup. I have outlined my decisions in a [thorough article](https://medium.com/homullus/docker-for-mac-performance-diy-d4m-e4232ca8b671).
 
 ### Naked
-You can just go oldschool and keep coding on your local machine. Use your editor of choice. All the files will be sync-ed over to the containers at nearly native speeds.
+Files - Host&VM; Editor - Host;
```
Comment (borisceranic) [inline README.md:34]: This looks to me almost like a broken HTML entity at a glance 😄 

Maybe you can improve formatting of this block using a table, to make it more obvious what are you trying to say?

Comment (borisceranic) [reply README.md:34]: @markomitranic ☝️

**Review by borisceranic — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (markomitranic) [reply README.md]: Hope this makes it better @borisceranic :)

Comment (borisceranic) [reply README.md]: Fantastic! You have unleashed your inner designer 😄

**Review by borisceranic — COMMENTED** [review]
Looks good!

**Review by borisceranic — APPROVED** [review]

**Pull request merged**